config/samples/config.yaml

```diff
@@ -0,0 +1,26 @@
+clusterName: "example-cluster"
+timeIntervalSec: 60
+repoCredentials:
+  "key1":
+    HTTPCredential:
+      Username: "exampleUser"
+      Password:
+        name: "http-creds"
+        key: "password"
+  "key2":
+    SSHCredential:
+      SSHKey:
+        name: "ssh-creds"
+        key: "sshKey"
+  "key3":
+    TLS:
+      insecureSkipVerify: true
+      CACertSecret:
+        name: "ca-cert-secret"
+        key: "ca.crt"
+      CertSecret:
+        name: "cert-secret"
+        key: "tls.crt"
+      KeySecret:
+        name: "key-secret"
+        key: "tls.key"
```

Comment:
So, I think you said something yesterday about actually testing with public/private key or something? Were you referring to TLS then?